packagingbuild/Dockerfile.debian

```diff
@@ -1,37 +1,50 @@
 
-# install python development
+{% if version in ('xenial') %}
```

Comment:
Looking at the history, the Dockerfiles always drifted too much and templating provided more hassle comparing to value it gave.

I'd prefer to remove the template files, focusing on just individual Dockerfiles.